### PR TITLE
CUMULUS-3165: Update example/cumulus-tf/orca.tf to orca v6.0.3

### PR DESCRIPTION
**Pull request by jennyhliu**
**Summary:** Summary of changes

Addresses [CUMULUS-3165: Update example/cumulus-tf/orca.tf to latest orca version](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3165)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-3147**
   - Set example/cumulus-tf default async_operation_image_version to 44.
   - Set example/cumulus-tf default ecs_task_image_version to 1.9.0.
+- **CUMULUS-3165**
+  - Update example/cumulus-tf/orca.tf to use orca v6.0.3
```
Comment (botanical) [inline CHANGELOG.md:28]: Can you move this to unreleased?

File: example/lambdas/python-reference-task/package.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ cp ./*.py ./dist/
 
 cd ./dist || exit 1
 
-find . -type f -print0 | xargs -0 node ../../../../bin/zip.js lambda.zip $(ls | grep -v lambda.zip)
```
Comment (botanical) [inline example/lambdas/python-reference-task/package.sh:23]: 👍

File: example/cumulus-tf/orca.tf [hunk 3]
```diff
@@ -38,7 +37,12 @@ module "orca" {
   db_admin_password    = local.rds_admin_login.password
   db_host_endpoint     = local.rds_admin_login.host
   db_user_password     = var.orca_db_user_password
+  dlq_subscription_email   = var.orca_dlq_subscription_email
   orca_default_bucket  = var.orca_default_bucket
+  orca_reports_bucket_name = var.system_bucket
+  rds_security_group_id    = local.rds_security_group
+  s3_access_key        = var.orca_s3_access_key
+  s3_secret_key        = var.orca_s3_secret_key
```
Comment (botanical) [inline example/cumulus-tf/orca.tf:45]: 👍

File: example/cumulus-tf/orca.tf [hunk 4]
```diff
@@ -23,7 +23,6 @@ module "orca" {
   lambda_subnet_ids        = local.subnet_ids
   permissions_boundary_arn = var.permissions_boundary_arn
   prefix                   = var.prefix
-  rds_security_group_id    = local.rds_security_group
```
Comment (botanical) [inline example/cumulus-tf/orca.tf:26]: Isn't this still a Cumulus variable? Why was it moved to ORCA variables?

**Review by botanical — CHANGES_REQUESTED** [review]
Had a question about the rds_security_group_id change and made one comment regarding moving the changelog entry

File: example/cumulus-tf/orca.tf (hunk 4 above)
Comment (jennyhliu) [reply example/cumulus-tf/orca.tf:26]: I followed the orca document: https://nasa.github.io/cumulus-orca/docs/developer/deployment-guide/deployment-with-cumulus

File: CHANGELOG.md (hunk 1 above)
Comment (jennyhliu) [reply CHANGELOG.md:33]: done

**Review by botanical — APPROVED** [review]
Thanks for addressing the feedback, looks great!

**Pull request merged**